pkg/scripts/poststart/check_fuse.go

```diff
@@ -57,6 +58,12 @@ do
     fi
 done
 
+# different with csi, as here the mount point is the parent dir of the fuse mount point, 
+if [ ! -e  $ConditionPathIsMountPoint/*/$SubPath ] ; then
+  echo "sub path [$SubPath] not exist!"
```

Comment:
ditto

Reply:
can only change to`echo "path [$ConditionPathIsMountPoint/*/$SubPath]  not exist!"` which seems wierd.